src/App.vue

```diff
@@ -1,8 +1,9 @@
 <template>
   <div id="app">
-    <router-view />
+    <transition :name="transitionName">
```

Comment:
Please change the transition name to something else than `transitionName` :stuck_out_tongue_winking_eye:

Reply:
Ah ok never mind, now I understand that this is catching the transition name from a variable named liked that hehehe, sorry!